eng/Versions.props

```diff
@@ -70,6 +70,52 @@
+    <MicrosoftExtensionsCachingMemoryLTSVersion>8.0.0</MicrosoftExtensionsCachingMemoryLTSVersion>
+    <MicrosoftExtensionsConfigurationAbstractionsLTSVersion>8.0.0</MicrosoftExtensionsConfigurationAbstractionsLTSVersion>
+    <MicrosoftExtensionsConfigurationBinderLTSVersion>8.0.2</MicrosoftExtensionsConfigurationBinderLTSVersion>
+    <MicrosoftExtensionsConfigurationJsonLTSVersion>8.0.0</MicrosoftExtensionsConfigurationJsonLTSVersion>
+    <MicrosoftExtensionsConfigurationLTSVersion>8.0.0</MicrosoftExtensionsConfigurationLTSVersion>
+    <MicrosoftExtensionsDependencyInjectionAbstractionsLTSVersion>8.0.1</MicrosoftExtensionsDependencyInjectionAbstractionsLTSVersion>
+    <MicrosoftExtensionsDependencyInjectionLTSVersion>8.0.0</MicrosoftExtensionsDependencyInjectionLTSVersion>
+    <MicrosoftExtensionsDiagnosticsLTSVersion>8.0.0</MicrosoftExtensionsDiagnosticsLTSVersion>
+    <MicrosoftExtensionsHostingAbstractionsLTSVersion>8.0.0</MicrosoftExtensionsHostingAbstractionsLTSVersion>
+    <MicrosoftExtensionsHostingLTSVersion>8.0.0</MicrosoftExtensionsHostingLTSVersion>
+    <MicrosoftExtensionsHttpLTSVersion>8.0.0</MicrosoftExtensionsHttpLTSVersion>
+    <MicrosoftExtensionsLoggingAbstractionsLTSVersion>8.0.1</MicrosoftExtensionsLoggingAbstractionsLTSVersion>
+    <MicrosoftExtensionsLoggingConfigurationLTSVersion>8.0.0</MicrosoftExtensionsLoggingConfigurationLTSVersion>
+    <MicrosoftExtensionsLoggingConsoleLTSVersion>8.0.0</MicrosoftExtensionsLoggingConsoleLTSVersion>
+    <MicrosoftExtensionsLoggingLTSVersion>8.0.0</MicrosoftExtensionsLoggingLTSVersion>
+    <MicrosoftExtensionsOptionsConfigurationExtensionsLTSVersion>8.0.0</MicrosoftExtensionsOptionsConfigurationExtensionsLTSVersion>
+    <MicrosoftExtensionsOptionsLTSVersion>8.0.2</MicrosoftExtensionsOptionsLTSVersion>
+    <MicrosoftNETCoreAppRefLTSVersion>8.0.8</MicrosoftNETCoreAppRefLTSVersion>
+    <MicrosoftNETCoreAppRuntimewinx64LTSVersion>8.0.8</MicrosoftNETCoreAppRuntimewinx64LTSVersion>
+    <SystemCollectionsImmutableLTSVersion>8.0.0</SystemCollectionsImmutableLTSVersion>
+    <SystemConfigurationConfigurationManagerLTSVersion>8.0.0</SystemConfigurationConfigurationManagerLTSVersion>
+    <SystemDiagnosticsDiagnosticSourceLTSVersion>8.0.1</SystemDiagnosticsDiagnosticSourceLTSVersion>
+    <SystemDiagnosticsPerformanceCounterLTSVersion>8.0.0</SystemDiagnosticsPerformanceCounterLTSVersion>
+    <SystemIOHashingLTSVersion>8.0.0</SystemIOHashingLTSVersion>
+    <SystemIOPipelinesLTSVersion>8.0.0</SystemIOPipelinesLTSVersion>
+    <SystemRuntimeCachingLTSVersion>8.0.0</SystemRuntimeCachingLTSVersion>
+    <SystemSecurityCryptographyPkcsLTSVersion>8.0.0</SystemSecurityCryptographyPkcsLTSVersion>
+    <SystemSecurityCryptographyXmlLTSVersion>8.0.1</SystemSecurityCryptographyXmlLTSVersion>
+    <SystemTextEncodingsWebLTSVersion>8.0.0</SystemTextEncodingsWebLTSVersion>
+    <SystemTextJsonLTSVersion>8.0.4</SystemTextJsonLTSVersion>
```

Comment:
8.0.5

Reply:
FWIW I just merged a PR that introduces the same version using a different name:

https://github.com/dotnet/extensions/pull/5513/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67R65

We should try to consolidate the two.

Reply:
We should have TFM-specific entries in different files. See https://github.com/dotnet/extensions/pull/5470#issuecomment-2398288981.

Reply:
Fixed